test/integration/001_simple_copy_test/test_simple_copy.py

```diff
@@ -154,3 +154,23 @@ def test__snowflake__seed__quoting_switch(self):
             "quoting": {"identifier": True},
         })
         results = self.run_dbt(["seed"], expect_pass=False)
+
+    @attr(type="bigquery")
+    def test__bigquery__simple_copy(self):
+        self.use_profile("postgres")
+        self.use_default_project({"data-paths": [self.dir("seed-initial")]})
+
+        self.run_dbt(["seed"])
+        self.run_dbt()
```

Comment:
These tests should probably check to make sure the right number of things are being run (see #854)